src/rules/trailingCommaRule.ts

```diff
@@ -230,4 +230,16 @@ class TrailingCommaWalker extends Lint.AbstractWalker<Options> {
             this.addFailureAt(list.end - 1, 1, Rule.FAILURE_STRING_NEVER, Lint.Replacement.deleteText(list.end - 1, 1));
         }
     }
+
+    private isWrappedSingleExpression(list: ts.NodeArray<ts.Node>) {
+        if (list.length !== 1) {
+            return false;
+        }
+
+        const parentNodeChildren = list[0].parent!.getChildren();
```

Comment:
For performance reasons (although barely noticeable), you should always pass the `sourceFile` as parameter to `.getChildren()`, `getStart()` and other methods.